tests/system/general/test_28_Maxwell3D.py

```diff
@@ -234,26 +234,26 @@ def test_06_eddycurrent(self):
 
     def test_07a_setup(self):
         adaptive_frequency = "200Hz"
-        Setup = self.aedtapp.create_setup()
-        Setup.props["MaximumPasses"] = 12
-        Setup.props["MinimumPasses"] = 2
-        Setup.props["MinimumConvergedPasses"] = 1
-        Setup.props["PercentRefinement"] = 30
-        Setup.props["Frequency"] = adaptive_frequency
+        setup = self.aedtapp.create_setup()
+        setup.props["MaximumPasses"] = 12
+        setup.props["MinimumPasses"] = 2
+        setup.props["MinimumConvergedPasses"] = 1
+        setup.props["PercentRefinement"] = 30
+        setup.props["Frequency"] = adaptive_frequency
         dc_freq = 0.1
         stop_freq = 10
         count = 1
-        assert Setup.add_eddy_current_sweep("LinearStep", dc_freq, stop_freq, count, clear=True)
-        assert isinstance(Setup.props["SweepRanges"]["Subrange"], dict)
-        assert Setup.props["SaveAllFields"]
-        assert Setup.add_eddy_current_sweep("LinearCount", dc_freq, stop_freq, count, clear=False)
-        assert isinstance(Setup.props["SweepRanges"]["Subrange"], list)
-
-        assert Setup.update()
-        assert Setup.enable_expression_cache(["CoreLoss"], "Fields", "Phase='0deg' ", True)
-        assert Setup.props["UseCacheFor"] == ["Pass", "Freq"]
-        assert Setup.disable()
-        assert Setup.enable()
+        assert setup.add_eddy_current_sweep("LinearStep", dc_freq, stop_freq, count, clear=True)
```

Comment:
Here I'm just being pedantic but maybe we can change the variable name from dc_freq to initial_freq, and count to freq_inc (for increment..or something along those lines). Note that the value of the increment is in Hz so the value count reads odd to me 

![image](https://github.com/user-attachments/assets/a76ef71c-03e9-4899-9b04-a9bc885e3f95)

Reply:
Even better, to have more consistency with HFSS.

start_frequency
stop_frequency

And sweep_type 
And step_size

Reply:
@jvela018 @Samuelopez-ansys agree, I will change this in the function handler